Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py

```diff
@@ -99,41 +98,44 @@ def PyExec(self):
         # Do setup
         self._setup()
 
-        logger.debug('in_ws:'+str(type(self._input_workspaces)))
-
         # Get input workspaces
         input_workspace_names = self._input_workspaces.getNames()
 
         # Lists of input and output workspaces
         q_workspaces = list()
         q2_workspaces = list()
         run_numbers = list()
-        temperatures = list()
+        sample_param = list()
+
+        progress = Progress(self, 0.0, 0.05, 3)
 
         # Perform the ElasticWindow algorithms
         for input_ws in input_workspace_names:
+
             logger.information('Running ElasticWindow for workspace: %s' % input_ws)
+            progress.report('ElasticWindow for workspace: %s' % input_ws)
 
             q_ws = '__' + input_ws + '_q'
             q2_ws = '__' + input_ws + '_q2'
 
+            elwin_alg = self.createChildAlgorithm("ElasticWindow", enableLogging=False)
+            elwin_alg.setAlwaysStoreInADS(True)
```

Comment:
It's good that we've switched to the `createChildAlgorithm` style. It's better practice if algorithms don't interact with the ADS themselves. Do we need this line?